src/js/components/Notification/Notification.js

```diff
@@ -45,39 +90,92 @@ const Notification = ({ message, onClose, id, status, title, toast }) => {
   ]);
 
   const { icon: CloseIcon } = theme.notification.close;
-  const { icon: StatusIcon, color } = theme.notification[status];
+  const { icon: StatusIcon, background, color } = theme.notification[status];
   const { color: closeIconColor } = theme.notification.close;
 
+  const { direction } = toast ? theme.notification.toast : theme.notification;
+
+  const TextWrapper = direction === 'row' ? Text : Fragment;
+
+  // notification is built with two child boxes that contain:
+  // 1. icon + text (wrapped in button when clickable)
+  // 2. close button
+  // pad needs to be applied to the child boxes, but we don't want to apply
+  // extra padding between the icon + text and the button.
+  const { pad } = theme.notification.container;
+  let textPad;
+  let closeButtonPad;
+  if (onClose) [textPad, closeButtonPad] = adaptThemeStyle(pad, theme);
+  else textPad = pad;
+
+  let actions;
+  let message = messageProp;
+
+  if (actionsProp)
+    actions = actionsProp.map((action, index) => (
+      <NotificationAnchor
+        key={action.label}
+        // create space between first anchor and text content
+        margin={
+          index === 0 && (message || title) ? { left: 'small' } : undefined
+        }
+        {...action}
+      />
+    ));
+
+  const MessageWrapper = direction !== 'row' ? Paragraph : Text;
```

Comment:
Wondering if we should just call this `Message`? Since we don't tend to think of "Text" being a "wrapper" for its string content.